-inputs.tf

```diff
@@ -1,34 +1,34 @@
 variable "approval_deploy_enabled" {
   description = "Enable approval for deployment step"
   type        = bool
-  default = true
+  default     = true
 }
 
 variable "artifact_store_bucket_name" {
   description = "Name of the bucket that stores artifacts"
-  type = string
+  type        = string
 }
 
 variable "codestar_connection_arn" {
   description = ""
-  type = string
+  type        = string
 }
 
 variable "description" {
   description = "Description of what your Lambda Function does."
-  type = string
+  type        = string
 }
 
 variable "detect_changes" {
   description = "Controls automatically starting your pipeline when a new commit is made on the configured repository and branch."
```

Comment:
I think this sentence is wrong, something like "Controls if the codepipeline execution is started automatically when you make a new commit on the repository".

Reply:
Perfect, I upgraded the description of the variable, thanks!